GPU/GLES/GLES_GPU.cpp

```diff
@@ -565,11 +565,21 @@ void GLES_GPU::CheckGPUFeatures() {
 	if (gl_extensions.GLES3 || !gl_extensions.IsGLES)
 		features |= GPU_SUPPORTS_TEXTURE_LOD_CONTROL;
 
-	// In the future, also disable this when we get a proper 16-bit depth buffer.
-	if ((!gl_extensions.IsGLES || gl_extensions.GLES3) && PSP_CoreParameter().compat.flags().PixelDepthRounding) {
-		features |= GPU_ROUND_FRAGMENT_DEPTH_TO_16BIT;
-	} else {
-		if (PSP_CoreParameter().compat.flags().VertexDepthRounding) {
+	// If we already have a 16-bit depth buffer, we don't need to round.
+	if (fbo_standard_z_depth() > 16) {
+		if (PSP_CoreParameter().compat.flags().PixelDepthRounding) {
+			if (!gl_extensions.IsGLES || gl_extensions.GLES3) {
+				// Use fragment rounding on desktop and GLES3, most accurate.
+				features |= GPU_ROUND_FRAGMENT_DEPTH_TO_16BIT;
+			} else if (fbo_standard_z_depth() == 24) {
+				// Here we can simulate a 16 bit depth buffer by clamping.
```

Comment:
I'm not sure I would call this clamping - rather just mapping to a reduce range. although CLAMP is shorter than anything I can come up with for the latter though :)